src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
+import InformationCard from "components/molecules/InformationCard";
+
+import "./WithInformationCard.scss";
+
+interface PropsType {
+  venue: AnyVenue;
+  showInfo?: boolean;
+}
+
+export const WithInformationCard: React.FunctionComponent<PropsType> = ({
```

Comment:
Export component props + name them as `[COMPONENTNAME]Props`:

```suggestion
export interface WithInformationCardProps {
  venue: AnyVenue;
  showInfo?: boolean;
}

export const WithInformationCard: React.FC<WithInformationCardProps> = ({
```